src/context/formsContext/FormsContext.ts

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import {YandexFormProps} from '../../models/constructor-items/common';
+import {HubspotFormProps} from '../../models/constructor-items/sub-blocks';
+
+export type FormsContextProps = YandexFormsContextProps | HubspotFormsContextProps;
```

Comment:
maybe we need to create `forms : {ya, hubspot}`, because users can have both of them ?

Reply:
done